release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1117,9 +1133,28 @@ If you are receiving pod security violations, see the following resources:
 
 * If necessary, you can set a custom admission profile on the namespace or pod by setting the `pod-security.kubernetes.io/enforce` label.
 
+[discrete]
+[id="ocp-4-12-olm-catalogs-and-psa-restricted-enforcement"]
+=== Catalog sources and restricted pod security admission enforcement
+
+Catalog sources built using the SQLite-based catalog format and a version of the `opm` CLI tool released before {product-title} 4.11 cannot run under restricted pod security enforcement.
+
+In {product-title} {product-version}, namespaces do not have restricted pod security enforcement by default and the default catalog source security mode is set to `legacy`.
+
+If you do not want to run your SQLite-based catalog source pods under restricted pod security enforcement, you do not need to update your catalog source in {product-title} {product-version}. However, if you do not take action to ensure your catalog sources run under restricted pod security enforcement, your catalog sources might not run in future {product-title} releases.
```

Comment:
This is looking good! I know ISG says to try and avoid negative constructions, so what do you think about this suggestion?
```suggestion
If you do not want to run your SQLite-based catalog source pods under restricted pod security enforcement, you do not need to update your catalog source in {product-title} {product-version}. However, to ensure your catalog sources run in future {product-title} releases, you must take action and update your catalog sources to run under restricted pod security enforcement.
```

Reply:
Suggestion applied. Thank you!